keyboards/custommk/ergostrafer_rgb/config.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 customMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// FRAM configuration
+#define EXTERNAL_EEPROM_SPI_SLAVE_SELECT_PIN B7
+#define EXTERNAL_EEPROM_SPI_CLOCK_DIVISOR 4 // 48MHz / 4 = 12MHz; max supported by MB85R64 is 20MHz
+
+// SPI configuration
+#define SPI_DRIVER SPID1
+#define SPI_SCK_PIN B3
+#define SPI_SCK_PAL_MODE 5
+#define SPI_MOSI_PIN B5
+#define SPI_MOSI_PAL_MODE 5
+#define SPI_MISO_PIN B4
+#define SPI_MISO_PAL_MODE 5
```

Comment:
Modes default to 5, so don't need to set.
```suggestion
#define SPI_MOSI_PIN B5
#define SPI_MISO_PIN B4
```